lookup.js

```diff
@@ -88,16 +89,7 @@ exports.init = function (sbot, config) {
     details.feedpurpose = content.feedpurpose || details.feedpurpose
     details.metafeed = content.metafeed || details.metafeed
     details.metadata = {} || details.metafeed
-    const NOT_METADATA = [
-      'metafeed',
-      'feedpurpose',
-      'type',
-      'tangles',
-      'reason',
-      'subfeed',
-      'nonce',
-    ]
-    const keys = Object.keys(content).filter((k) => !NOT_METADATA.includes(k))
+    const keys = Object.keys(content).filter((k) => !NOT_METADATA.has(k))
```

Comment:
why not includes here? It seems to me that has in not an options on array (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) and thus operates on object that just happens to work? Or am I missing something?

Reply:
Ah, did not see that. Perfect